File: templates/python/es/text/code/beeperLineBug1.py
# Archivo: Cono LineBug.py
# ------------------------------
# Utiliza ciclo while para colocar una línea de conos .
# Este programa funciona para un mundo de cualquier tamaño.
# Sin embargo, porque cada mundo requiere uno menos
# moverse s que put_ conos siempre falta un cono .
from karel.stanfordkarel import *

# el programa comienza en main
def main():
   # se repite hasta que karel enfrenta a una pared
   while frente_despejado():
      # colocar un cono en el cuadrado actual
      poner_cono()
      # moverse al siguiente cuadrado
      moverse()

File: templates/java/es/text/code/beeperLineBug1.java
/*
 * Archivo: ConoErrorDeLínea.java
 * -----------------------------
 * Utiliza un ciclo while para colocar una línea de c
 * onos . Este programa funciona para un mundo de cua
 * lquier tamaño. Sin embargo, debido a que cada mund
 * o requiere uno menos moverserse s que ponerCono s, si
 * empre se pierde un cono .
 */
import stanford.karel.*;
public class ConoErrorDeLínea extends Karel {
   public void run() {
   	  /* Se repite hasta karel frente a una pared. */
      while(frenteDespejado()) {
      	 /* coloca un cono en la plaza actual */ 
         ponerCono();
      	 /* moverse al siguiente cuadrado */
         moverse();
      }
   }
}